README.md

```diff
@@ -151,6 +151,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 **Rust**
 - [QCGPU](https://github.com/QCGPU/qcgpu-rust) - High-performance GPU-accelerated quantum computer simulation outlined in this [arXiv paper](https://arxiv.org/pdf/1805.00988.pdf).
 - [RustQIP](https://github.com/Renmusxd/RustQIP) - Rust Quantum Computing library leveraging graph building to build efficient quantum circuit simulations.
+- [Quriust](https://github.com/ScipioneParmigiano/) - A blazing fast Rust library for simulating quantum circuits. Only for quriust ones.
```

Comment:
This should be in alphabetical order.

Reply:
Also, wrong link? Should be: https://github.com/ScipioneParmigiano/quriust

Reply:
Correct, thx 👍